.github/workflows/deploy-oss-catalog.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy OSS Connector Catalog to GCS
+
+on:
+  push:
+    branches:
+      - master
+    paths: 
+      - airbyte-config/init/src/main/resources/seed
+
+  workflow_dispatch:
+
+jobs:
+  deploy-catalog:
+    name: "Deploy Catalog"
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    concurrency: deploy-oss-connector-catalog
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - name: Setup Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0
+        with:
+          service_account_key: ${{ secrets.PROD_SPEC_CACHE_SA_KEY }}
```

Comment:
Added this new secret after chatting with @git-phu, who verified the `prod-spec-cache-bucket-SA-key` from LastPass has access to the CMS bucket